src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.datasource.url=jdbc:postgresql://localhost:5432/innowise
```

Comment:
I have a problem with connecting to postgres container from outside (It tells me to check if the port accept tcp/ip)